src/main/java/com/robosh/ejournal/service/AdminService.java

```diff
@@ -28,7 +28,6 @@ public AdminInfoDto save(SaveAdminDto saveAdminDto) {
         Admin admin = adminMapper.fromSaveAdminDtoToAdmin(saveAdminDto);
 
         saveSchoolForAdmin(saveAdminDto, admin);
-
         ValidatorProcessor.validate(admin);
         adminRepository.save(admin);
         log.info("Admin saved");
```

Comment:
maybe we need put admin object into log? need to discuss

Reply:
It's not secure. As we can see a password

Reply:
just transform it to info dto, ez

Reply:
I think it's too many work to create only one log. Also a lot of logs not good